interactive_engine/distribution/src/conf/log4rs.yml

```diff
@@ -1,109 +1,27 @@
-refresh_rate: 10 seconds
-
+refresh_rate: 30 seconds
 appenders:
   stdout:
     kind: console
     encoder:
       pattern: "{d(%Y-%m-%d %H:%M:%S.%f)} {h({l:<5})} (({f}:{L})) [{T}] {m}{n}"
-  executor:
-    kind: env_rolling_file
+  file:
+    kind: rolling_file
     append: true
-    path: "${env:LOG_DIRS:-./}/executor.log"
+    path: "/tmp/graphscope-store/rust.log"
```

Comment:
why renames this filename?
I think `executor.log` is better than `rust.log`, it is more self-described.

Reply:
I not sure if this is the log for executor ? @tianliplus Can you give a reply comment here ?